news/_posts/2020-12-09-pr-testing-on-prestashop.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: post
+title:  "PR Testing on PrestaShop"
+subtitle: ""
+date:   2020-12-08 16:00:00
+authors: [ RobinFischer ]
+icon: icon-airplane
+tags:
+ - 
+---
```

Comment:
Little trick: by ysing "View file" on the .md file you can see how it will be rendered
https://github.com/PrestaShop/prestashop.github.io/blob/5f6a8256407f368241ce69d2cf23e265a35c6340/news/_posts/2020-12-09-pr-testing-on-prestashop.md